buildSrc/src/main/kotlin/com/edricchan/studybuddy/library/android-compose.gradle.kts

```diff
@@ -0,0 +1,11 @@
+/** Convention plugin for Android libraries made with Jetpack Compose. */
+package com.edricchan.studybuddy.library
+
+plugins {
+    id("com.edricchan.studybuddy.library.android")
+}
+
+android {
+    buildFeatures.compose = true
+    composeOptions.kotlinCompilerExtensionVersion = "1.4.7"
```

Comment:
version catalog? :eyes:

Reply:
Version catalogs aren't natively supported in convention plugins AFAICT - see gradle/gradle#15383